pgo-rmdata/rmdata/process.go

```diff
@@ -658,14 +658,38 @@ func removeBackupJobs(request Request) {
 // "rm -rf" like in other commands). Well, we could...we could write a job to do
 // this, but that will be saved for future work
 func removeLogicalBackupPVCs(request Request) {
-	// get the name of the PVC, which uses a format that is fixed
-	pvcName := fmt.Sprintf(pgDumpPVC, request.ClusterName)
 
-	log.Debugf("remove pgdump pvc name [%s]", pvcName)
+	pvcList := make([]string, 0)
+	selector := fmt.Sprintf("%s=%s", config.LABEL_PG_CLUSTER, request.ClusterName)
+	dumpPrefix := fmt.Sprintf(pgDumpPVCPrefix, request.ClusterName)
+
+	// get all of the PVCs to analyze (see the step below)
+	pvcs, err := request.Clientset.
+		CoreV1().PersistentVolumeClaims(request.Namespace).
+		List(metav1.ListOptions{LabelSelector: selector})
+	if err != nil {
+		log.Error(err)
+		return
+	}
+
+	// Now iterate through all the PVCs to identify those that are for a logical backup and add
+	// them to the PVC list for deletion.  This pattern matching will be utilized until better
+	// labeling is in place to uniquely identify logical backup PVCs.
+	for _, pvc := range pvcs.Items {
+		pvcName := pvc.GetName()
+
+		if !strings.HasPrefix(pvcName, dumpPrefix) {
+			log.Debug("skipping...")
+			continue
+		}
+
+		log.Debugf("found logical backup pvc: [%s]", pvcName)
```

Comment:
I think the `log.Debug()` call on L691 is enough. The two calls inside the loop are just noise.

Reply:
I copied those debug statements from some similar contributions in `rmdata` :wink:

https://github.com/CrunchyData/postgres-operator/blob/110bd433d2217d2e47a6c46e406790088313cc4c/pgo-rmdata/rmdata/process.go#L506-L509

Definitely agree they aren't needed tho, so went ahead and removed them.  Thanks!